lib/widgets/profile.dart

```diff
@@ -47,6 +48,7 @@ class ProfilePage extends StatelessWidget {
     if (user == null) {
       return const _ProfileErrorPage();
     }
+    final userStatus = store.getUserStatus(userId);
```

Comment:
nit: put this next to `displayEmail`; conceptually they're here for the very same reason